fact#28: add connection between 2 DBs

PeindreLaColline opened the pull request:
# PR Description
> Redis, MySQL 동기화 코드 추가
현재 로직: redis에 조회수를 string으로, 조회된 게시글의 postId를 set으로 저장해서 10초에 한 번씩 MySQL에 업데이트 하도록 했습니다.
수정 계획: set에 저장되는 postId의 개수 제한이 현재 없습니다. 일정 개수가 넘으면 오래된 순으로 지워지게 수정할 예정입니다. 
(기타 사항: 쿠키 포함 테스트 완료)

## Related Issue

close #28

## Help
> 잠시만 시간을,,

PeindreLaColline committed:
fact: add connection between 2 DBs

PeindreLaColline committed:
fact: add connection between 2 DBs

hyewonbae commented on src/main/java/mini/backend/config/SecurityConfig.java:
요 부분 테스트 때문에 변경했었던 걸까요?

hyewonbae commented on src/main/java/mini/backend/config/SecurityConfig.java:
```diff
@@ -39,8 +39,8 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
                 .authorizeHttpRequests(authorize -> authorize
-                        .requestMatchers(AUTH_WHITELIST).permitAll()
-                        .anyRequest().authenticated()
+                        //.requestMatchers(AUTH_WHITELIST).permitAll()
+                        .anyRequest().permitAll()
```
요 부분은 테스트 때문에 변경하신 걸까요?

PeindreLaColline replied:
넵 브라우저에서 로그인창 자꾸 떠서 수정했습니다 커밋을 같이해버렸네요
감사합니닷

hyewonbae committed:
Feat/#23 post like (#29)

* feat : create post like

- 게시물 좋아요 기능구현
+ refact : move entity classes within domain package
- 도메인 패키지 안의 엔티티 클래스 이동 리팩토링

* feat : add sync logic

동기화 로직 추가 및 리팩토링

* fix : Solving FK Constraint Issues

1. post (부모)테이블에서 레코드 삭제 시 참조하는 comment (자식)테이블 레코드 선삭제

2. Like entity naming
snake case -> camel case

PeindreLaColline committed:
Merge branch 'feat#28-add-connection-DB-postView' of https://github.com/URECA-mini4/backend into feat#28-add-connection-DB-postView

PeindreLaColline committed:
fact: add delete oldest postId in Redis

PeindreLaColline commented:
수정계획에 적어두었던 부분들까지 모두 개발 완료했습니다

weejinyoung commented:
post:{postId} 
이런 키를 가진 Set 자료구조로 구현하는 방법도 꼭 고민해보시길 바라요

PeindreLaColline committed:
feat: fix proxy internal call

PeindreLaColline commented:
> post:{postId} 이런 키를 가진 Set 자료구조로 구현하는 방법도 꼭 고민해보시길 바라요

넵 알겠습니다!
